### ISSUE TITLE
[Feature Request] Additional ppsspp shaders

### ISSUE THREAD

**Issue by OrangeSherbert** [cizia64/CrossMix-OS#287]
Some of the lightweight mobile shaders in this repo can improve the look of psp games that will only run at 1x resolution.  May be worth including them along with the default ppsspp ones?

https://github.com/jdgleaver/ppsspp_shaders

[lcd1x_sharp+psp_color_mobile.fsh](https://github.com/jdgleaver/ppsspp_shaders/blob/master/lcd1x_sharp%2Bpsp_color_mobile.fsh)
[lcd1x_sharp_mobile.fsh](https://github.com/jdgleaver/ppsspp_shaders/blob/master/lcd1x_sharp_mobile.fsh)
[lcd3x_mobile.fsh](https://github.com/jdgleaver/ppsspp_shaders/blob/master/lcd3x_mobile.fsh)
[lcd3x+psp_color_mobile.fsh](https://github.com/jdgleaver/ppsspp_shaders/blob/master/lcd3x%2Bpsp_color_mobile.fsh)
[psp_color_mobile.fsh](https://github.com/jdgleaver/ppsspp_shaders/blob/master/psp_color_mobile.fsh)

**Comment by cizia64** [comment]
Thanks, this file list is the one you recommend to add ?
Do you have some screenshot of what we can expect as results ?

**Comment by OrangeSherbert** [comment]
Yeah those are the ones I recommend. There is an vsh, fsh and ini file for each one that just needs to be dropped into the ppsspp shader folder. Here’s a screenshot for comparison.

No shader:
[image0.jpeg]

LCD3x + PSP Color (Mobile)
[image1.jpeg]



On Dec 12, 2024, at 5:06 PM, cizia64 ***@***.***> wrote:

﻿

Thanks, this file list is the one you recommend to add ?
Do you have some screenshot of what we can expect as results ?
